thunder/executors/apex_entropyex_impl.py

```diff
@@ -239,10 +243,11 @@ def _apex_cross_entropy_grad(
     return fwd
 
 
-# Registers the implementation for torch.nn.functional.cross_entropy
-apex_ex.register_implementation(
-    ltorch.cross_entropy,
-    checker=_cross_entropy_checker,
-    execution_transform=_cross_entropy_transform,
-    grad_transform=_apex_cross_entropy_grad,
-)
+if apex_entropy_available():
```

Comment:
Alternatively, `apex_entropy_available()` could be called inside the checker function.

Reply:
I was thinking since the op won't be available, it would be better to not register at all instead of this happening later in checker.